### ISSUE TITLE
Add launchpad descriptions for rockets

### ISSUE THREAD

**Issue by HaHaWTH** [igentuman/GalacticResearch#11]
Please add a description that shows what launchpad the rocket need like GalactiCraft,
this will make players easier to understand,thanks

**Comment by igentuman** [comment]
Can you rephrase please. I don't understand

**Comment by HaHaWTH** [comment]
> Can you rephrase please. I don't understand

Add the descriptions like this
![this](https://user-images.githubusercontent.com/102713261/219857244-1d43f7c2-5eb7-492e-8cf9-225f17a8762d.png)
There are many kinds of launchpads in Galacticraft and its addons,add a description will makes it clear to know which launchpad does the rocket use

**Comment by igentuman** [comment]
Instead of adding toltips i will make rockets compatible with any kind of pads

**Comment by igentuman** [comment]
Well, i've managed to add compatibility with EP and GS landing pad. But for some weird reasons rockets can't launch from these pads. So i've added tooltips for the rockets
